database/supply.go

```diff
@@ -16,3 +16,27 @@ func (db BigDipperDb) SaveSupplyToken(coins sdk.Coins, height int64) error {
 	}
 	return nil
 }
+
+//GetTokenNames get token name from  latest height
+func (db BigDipperDb) GetTokenNames() ([]string, error) {
+	var names []string
+	query := `SELECT (coin).denom FROM (
+        SELECT unnest(coins) AS coin FROM supply WHERE height = (
+            SELECT max(height) FROM supply
+            ) 
+		) AS unnested`
+	if err := db.Sqlx.Select(&names, query); err != nil {
+		return nil, err
+	}
+	return names, nil
+}
+
+//return the latest height(has 30 second latency because depend on supply module)
+func (db BigDipperDb) GetLatestHeight() (int64, error) {
+	var height []int64
+	query := `select max(height) FROM supply`
+	if err := db.Sqlx.Select(&height, query); err != nil {
+		return -1, err
+	}
+	return height[0], nil
+}
```

Comment:
I think it's better to remove this. If we want to have the latest block height for some reason, it's better to use a `BlockHandler` instead that always has the updated one

Reply:
I actually don't know if saving block height or timestamp will be better for this case. Maybe just storing timestamps is better?

Reply:
@HarleyAppleChoi Yeah, I think timestamp would be better in this case